service/confirmation-repeater/src/index.ts

```diff
@@ -4,17 +4,22 @@ const amqplib = require('amqplib');
 const { Level } = require("level");
 const schedule = require('node-schedule');
 //const nodeSchedule = require("@types/node-schedule")
-const db = new Level(process.env.DB_PATH || "./db", { valueEncoding: 'json' })
-
-const dotenv = require('dotenv');
-dotenv.config();
+const db = new Level(process.env.DB_PATH || "./repeater.db", { valueEncoding: 'json' })
 
 const user = process.env.RABBIT_USER;
 const pass = process.env.RABBIT_PASSWORD;
 const queueConfirm = process.env.CONFIRM_QUEUE || "";
 const queueConfirmed = process.env.CONFIRMED_QUEUE || "";
-const maxRetries = process.env.MAX_RETIRES || "3";
+const maxRetries = parseInt(process.env.MAX_RETIRES || "3");
+
+const debugDayOffset = parseInt(process.env.ADD_DAYS || "0")
+
 
+type LevelError = {
```

Comment:
if Level package does not export error type, add it yourself.. later cast exception to it - see below